content/documentation/overview/index.adoc

```diff
@@ -116,11 +116,10 @@ image::istio-yaml-validation.png[Demo Website]
 
 === Distributed Tracing
 
-In the distributed tracing tab we can see the Jaeger UI in a new tab to check the traces generated by our resources.
-
+The Distributed Tracing menu will open the whole https://www.jaegertracing.io/[Jaeger] UI in a new tab for tracing services.
```

Comment:
"Clicking on Distributed Tracing menu item will open a new tab with the https://www.jaegertracing.io/[Jaeger] UI for tracing services"